test/e2e/storage/drivers/csi-test/driver/driver.mock.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The Kubernetes Authors.
+Copyright The Kubernetes Authors.
```

Comment:
Is this line manually changed or automatically generated?

Reply:
@xing-yang Thank You very much for the review

the copyright file will be added automatically by the `update-mocks.sh` script.

it will use the template from `hack/boilerplate/boilerplate.generatego.txt`

https://github.com/kubernetes/kubernetes/blob/1895777d6cf5e40bc4b90f5a15a7d3d022e3f581/hack/update-mocks.sh#L98-L107

Reply:
also the reason for removing YEAR from copyright text is added here https://github.com/kubernetes/kubernetes/pull/103631#issuecomment-907726028